.github/workflows/tests-types.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
```

Comment:
maybe we can do the tests/types stuff in a separate pr too, and we can land this one quickly.